src/relayclient/RelayClient.ts

```diff
@@ -268,9 +268,16 @@ export default class RelayClient {
   async resolveForwarder (gsnTransactionDetails: GsnTransactionDetails): Promise<Address> {
     let forwarderAddress = gsnTransactionDetails.forwarder ?? this.config.forwarderAddress
     if (forwarderAddress !== constants.ZERO_ADDRESS) {
-      const isTrusted = await this.contractInteractor.isTrustedForwarder(gsnTransactionDetails.to, forwarderAddress)
-      if (!isTrusted) {
-        throw new Error('The Forwarder address configured but is not trusted by the Recipient contract')
+      const recipientCode = await web3.eth.getCode(gsnTransactionDetails.to)
```

Comment:
This is adding a paymaster-specific code into the core RelayRecipient code.

this is duplicating a test that should be in paymaster (in paymasters repository) into the core GSN package.